sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/TracezZPageHandler.java

```diff
@@ -486,18 +486,19 @@ private void emitHtmlBody(Map<String, String> queryMap, PrintStream out)
               // Display latency based span
               LatencyBoundary latencyBoundary = LatencyBoundary.values()[subtype];
               spans =
-                  dataAggregator.getOkSpans(
-                      spanName,
-                      latencyBoundary.getLatencyLowerBound(),
-                      latencyBoundary.getLatencyUpperBound());
+                  new ArrayList<>(
+                      dataAggregator.getOkSpans(
+                          spanName,
+                          latencyBoundary.getLatencyLowerBound(),
+                          latencyBoundary.getLatencyUpperBound()));
               Collections.sort(spans, new SpanDataComparator(/* incremental= */ false));
             } else {
               if (subtype < 0 || subtype >= CanonicalCode.values().length) {
                 // N/A or out-of-bound cueck for error based subtype, valid values: [0, 15]
                 return;
               }
               // Display error based span
-              spans = dataAggregator.getErrorSpans(spanName);
+              spans = new ArrayList<>(dataAggregator.getErrorSpans(spanName));
               Collections.sort(spans, new SpanDataComparator(/* incremental= */ false));
```

Comment:
you can pull this sort call up one level, right? Since it's the same in both the if and the else branches?

Reply:
Hmm yes, I think originally it's meant for sorting in a different order, but now it's all the same, I'll pull them out